tests/ocf-zip-comp/EPUB/package.opf

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Zheng Xu</dc:creator>
+    <dc:date>2022-07-30</dc:date>
+        <dc:description>MUST treat any OCF ZIP container that uses compression techniques other than Deflate as in error.</dc:description>
+    <dc:identifier id="pub-id">ocf-zip-comp</dc:identifier>
```

Comment:
Minor knit: can you align the entry to the left with the rest of the items? Thx.

Reply:
thnx, updated